### PR TITLE
MBS-8599 fix scroll problem (closing soft keyboard)

### PR DESCRIPTION
**Pull request by dsvoronin**

### CODE REVIEW

File: subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/element/field/TextFieldAction.kt [hunk 1]
```diff
@@ -41,9 +41,9 @@ internal class TextFieldAction(
                 TypeText(
                     stringToBeTyped = text
                 )
-            )
+            ),
+            closeSoftKeyboard()
```
Comment (eugene-krivobokov) [inline subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/element/field/TextFieldAction.kt:45]: Can we reproduce the problem in a test?

**Review by eugene-krivobokov — APPROVED** [review]

File: subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/element/field/TextFieldAction.kt (hunk 1 above)
Comment (dsvoronin) [reply subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/element/field/TextFieldAction.kt:45]: will try

File: subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/element/field/TextFieldAction.kt [hunk 2]
```diff
@@ -57,9 +57,9 @@ internal class TextFieldAction(
                     stringToBeTyped = text
                 )
             ),
-            pressImeActionButton()
+            pressImeActionButton(),
```
Comment (sergioboi) [inline subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/element/field/TextFieldAction.kt:60]: So what is changed? I didn't get the idea of that fix
Is the difference between closeSoftKeyboard or we must call pressImeActionButton differently?

Comment (dsvoronin) [reply subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/element/field/TextFieldAction.kt:60]: no, closeSoftKeyboard was called without waitFor

**Pull request merged**